deploy/skaffold/Dockerfile

```diff
@@ -14,8 +14,12 @@
 
 FROM gcr.io/gcp-runtimes/ubuntu_16_0_4 as runtime_deps
 
+# needed for dateTime tagger
+COPY --from=golang:1.11 /usr/local/go/lib/time/zoneinfo.zip /usr/local/go/lib/time/zoneinfo.zip
```

Comment:
hmmm...this smells a little bit for me...and not just me: https://github.com/golang/go/issues/21881#issuecomment-370125690 - maybe we should try 4d63.com/tz ?

Reply:
Yes, of course! With `4d63.com/tz` the dateTime tagger will also work on windows :)